### DISCUSSION TITLE
How does the `input version` work?

### DISCUSSION THREAD

**Discussion by AsheKR** [release-drafter/release-drafter#1030] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42999044/150171815-58b2394e-4736-4c67-bbe2-0d7f758efab5.png)

Original version: `1.0.0`
Next version: `1.0.1`

If I set the version to `2.0.0` here, should it be as below?
Next version: `2.0.0`

I wanted this to happen, but it didn't work.
How does the version work? Please explain it in more detail.

**Comment by AsheKR** [comment, marked as answer]
I was confused because I didn't change tag and name. It's been solved.

**Comment by jetersen** [comment]
You need to use the template variable `$RESOLVED_VERSION` or `$INPUT_VERSION`.

`$RESOLVED_VERSION` uses  `$INPUT_VERSION` if `version` is provided to the action otherwise it uses semver to increment to an resolved version based on parameters for `version-resolver` config

**Reply by Ismoh** [reply]
Is there a way to get the version by reading a file?
I have a SemVer file already and need to keep it, because it's used to show the version in my mod in game!
